libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -915,6 +915,10 @@ namespace SohImGui {
                     Tooltip("Allows the Lon Lon Ranch obstacle course reward to be shared across time periods");
                     EnhancementCheckbox("Enable visible guard vision", "gGuardVision");
                     EnhancementCheckbox("Enable passage of time on file select", "gTimeFlowFileSelect");
+                    EnhancementCheckbox("No Random Drops", "gNoRandomDrops");
+                    Tooltip("Disables random drops, except from the Goron Pot, Dampe, and bosses");
+                    EnhancementCheckbox("No Heart Drops", "gNoHeartDrops");
+                    Tooltip("Disables heart drops, but not heart placements, like from a Deku Scrub running off");
```

Comment:
```suggestion
                    Tooltip("Disables heart drops, but not heart placements, like from a Deku Scrub running off. This simulates Hero Mode from other games in the series.");
```

Reply:
I mean, it simulates the "no heart drops" aspect of Hero Mode, not Hero Mode in its entirety, but sure. I'll make that change regardless.